pandas/core/internals/construction.py

```diff
@@ -577,6 +577,9 @@ def sanitize_array(data, index, dtype=None, copy=False,
             # we will try to copy be-definition here
             subarr = _try_cast(data, True, dtype, copy, raise_cast_failure)
 
+    elif isinstance(data, ABCNumPyExtensionArray):
```

Comment:
why is this ABCNumPyExtensionArray and not ABCPandasArray?

Reply:
Leftover from the earlier name in this PR. I can rename it if you want. But what's clearer at a glance? To me `ABCPandasArray` sounds kind of like anything that can be stored in a pandas container (i.e. `ExtensionArray`). I hoped that `ABCNumPyExtensionArray` is a bit clearer.